### PR TITLE
[Docs] Qdrant Cloud API Specification - Rough Overview

### PR DESCRIPTION
**Pull request by davidmyriel**
This is just a basic overview with a link. We need to do more work on the actual json spec file itself. 

[PREVIEW](https://deploy-preview-1119--condescending-goldwasser-91acf0.netlify.app/documentation/qdrant-cloud-api/)

### CODE REVIEW

**Comment by mikheillomidzeq** [comment]
Since Cloud API can manage Hybrid and managed clusters, I would put it one level up. 

Screenshot 2024-08-22 at 09 51 15

File: qdrant-landing/content/documentation/qdrant-cloud-api.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+title: Qdrant Cloud API
+weight: 10
+---
+# Qdrant Cloud API 
+
+The Qdrant Cloud API lets you manage Cloud accounts and their respective Qdrant clusters. You can use this API to manage your clusters, authentication methods, and cloud configurations. 
+
+| REST API      | Documentation                                                                        |
+| -------- | ------------------------------------------------------------------------------------ |
+| v.0.1.0 | [OpenAPI Specification](https://cloud.qdrant.io/pa/v1/docs/)                       |
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/qdrant-cloud-api.md:11]:
```suggestion
| v.0.1.0 | [OpenAPI Specification](https://cloud.qdrant.io/pa/v1/docs)                       |
```

This must be `https://cloud.qdrant.io/pa/v1/docs` without the trailing slash.

File: qdrant-landing/content/documentation/qdrant-cloud-api.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Cluster Authentication Management
+Use these endpoints to manage your cluster API keys.
+   - **List API Keys**: Retrieve all API keys associated with an account.
+   - **Create API Key**: Generate a new API key for programmatic access.
+   - **Delete API Key**: Revoke access by deleting a specific API key.
+   - **Update API Key**: Modify attributes of an existing API key.
+
+## Booking Packages
+You can manage available service packages across various cloud providers (AWS, GCP, Azure) and their respective regions. Use these endpoints to customize your deployments based on specific needs.
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/qdrant-cloud-api.md:52]: I don't understand this, and I'd assume also customers would struggle with this:

* What is a booking package
* What is a deployment in this context
* Why is this paragraph needed at all

I assume with "deployment" you mean a qdrant database cluster. And with "booking package" the configuration of this cluster.

I would add this then in simpler words to the Cluster Management section see above

File: qdrant-landing/content/documentation/qdrant-cloud-api.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+curl -X 'GET' \
+  'https://cloud.qdrant.io/pa/v1/accounts/<YOUR_ACCOUNT_ID>/clusters' \
+  -H 'accept: application/json' \
+  -H 'Authorization: <YOUR_API_KEY>'
+```
+
+This request will return a list of clusters associated with your account in JSON format.
+
+## Cluster Management  
+In addition to basic CRUD operations, the API supports fine-grained control over cluster resources (CPU, RAM, disk), node configurations, tolerations, and other operational characteristics.
```
Comment (bashofmann) [inline qdrant-landing/content/documentation/qdrant-cloud-api.md:37]:
```suggestion
Use these endpoints to create and manage your Qdrant database clusters. The API supports fine-grained control over cluster resources (CPU, RAM, disk), node configurations, tolerations, and other operational characteristics across all cloud providers (AWS, GCP, Azure) and their respective regions in Qdrant Cloud, as well as Hybrid Cloud. 
```

**Pull request merged**